clang/docs/ReleaseNotes.rst

```diff
@@ -488,8 +489,11 @@ Removed Compiler Flags
   derivatives) is now removed, since it's no longer possible to suppress the
   diagnostic (see above). Users can expect an `unknown warning` diagnostic if
   it's still in use.
-- The experimental flag '-ubsan-unique-traps' has been removed. It is
-  superseded by '-fno-sanitize-merge'.
+- The experimental flags '-ubsan-unique-traps' and
```

Comment:
this is -mllvm flag, they don't worth mentioning

Reply:
Removed